Woof/Woof/ViewModel/SitterProfileViewModel.swift

```diff
@@ -52,7 +52,7 @@ final class SitterProfileViewModel: ObservableObject {
 
     // MARK: - Private interface
 
-    private lazy var currentSitter: Sitter = loadSitterFromStorage()
+    private(set) lazy var currentSitter: Sitter = loadSitterFromStorage()
```

Comment:
```suggestion
    private lazy var currentSitter: Sitter = loadSitterFromStorage()
```
Why do we need `set` here?

Reply:
Now we don't need it.
Removed.